codelabs/gRPC_Cpp_OpenTelemetry_Plugin/completed/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+# Copyright 2018 gRPC authors.
```

Comment:
```suggestion
# Copyright 2024 gRPC authors.
```

Reply:
no, we're not supposed to change the years since i'm reusing content